go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -620,11 +620,13 @@ func (m leaderMetrics) Off(string) {
 	}
 }
 
+func (m leaderMetrics) SlowpathExercised(string) {}
+
```

Comment:
@martinkennelly this could potentially take a metric, seems like some kind of slow path leader acquire.